schema/dataKeys.json

```diff
@@ -136,12 +136,8 @@
           "description": "The product stream e.g. RHEL-tp1"
         },
         "cpe": {
-          "type": ["array", "string"],
-          "minItems": 1,
-          "description": "The list of product CPE IDs e.g. [\"cpe:/a:example:openstack:el8\"]",
-          "items": {
-            "type": "string"
-          }
+          "type": "string",
```

Comment:
Just curious, it seems we never did anything in the template: https://github.com/konflux-ci/release-service-utils/blob/main/templates/advisory.yaml.jinja#L12

So I guess it worked with both? That seems likely - the array would just be spitted there. But we have that Jira to wrap the values with `""` and at that point having an array would be an issue.

Just wondering if you thought of that at all. This change looks ok.

Reply:
Nope, I never thought about that. Good call out, but yeah shouldn't really matter now that we aren't going with it. We *did* proceed in this direction with product_id, but that is just an int (not string) so no quotes and seemingly no problem (e2e was already changed and things are going fine)

Reply:
OK, cool.